backend/server.ts

```diff
@@ -28,10 +29,10 @@ const storage = multer.memoryStorage(); // Store the file in memory
 const upload = multer({ storage: storage });
 
 app.get('/upload-page', (req, res) => {
-    res.sendFile(path.join(__dirname, '../Frontend/website.html'));
+    res.sendFile(path.join(__dirname, '../Frontend/testwebsite.html'));
 });
 
-app.post('/upload', upload.single('packageContent'), (req, res, next) => {
+app.post('/upload', upload.single('packageContent'), async (req, res, next) => {
```

Comment:
having the "next" field seems a little unnecessary since it's just doing a res.status(500).send("Internal Server Error").
Also, I would change console.log to a logger statement just for the future (since we'll have to log the inputs/outputs from the auto grader).

Reply:
Got rid of next and just added res.status(500).send('Internal Server Error');